nixos/modules/services/system/earlyoom.nix

```diff
@@ -63,6 +63,16 @@ in
           Enable debugging messages.
         '';
       };
+
+      notificationsCommand = mkOption {
+        type = types.nullOr types.str;
+        default = null;
+        example = "sudo -u example_user DISPLAY=:0 DBUS_SESSION_BUS_ADDRESS=unix:path=/run/user/1000/bus notify-send";
+        description = ''
+          Command used to send notifications.
+          See https://github.com/rfjakob/earlyoom#notifications for details.
```

Comment:
Oh one more thing, you need to wrap the link in `<link xlink:href="https://..."/>` to make it show up correctly in the rendered docs.

Reply:
done